content/baseQuote.ttl

```diff
@@ -2,8 +2,14 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 :baseQuote
     skos:prefLabel "base-quote"@en;
+    skos:related "forex";
```

Comment:
Put option fields below mandatory ones

Reply:
You mean to say, below skos:definition?

Reply:
Yes

Reply:
> Put option fields below mandatory ones

This has the feeling of something that should be left up to linting/pre-commit checks/code-formatting or something to be resolved in presentation, i.e. as data it shouldn't be important how it is ordered.

There may be other solutions:

* comment in the template,
* pr submission template, asking for contributors to check this,
* something else.

NB. were I reviewing this PR, I'm probably not picking this up.

Reply:
For now, contributors should adhere to the line length presents. Auto-enforcement through linting would be cool. I have manually adjusted what exists... this should be less of an issue going forward